rules/REQUEST-932-APPLICATION-ATTACK-RCE.conf

```
@@ -433,6 +433,40 @@ SecRule REQUEST_COOKIES|REQUEST_COOKIES_NAMES|ARGS_NAMES|ARGS|XML:/* "@rx \{[0-9
     setvar:'tx.inbound_anomaly_score_pl1=+%{tx.critical_anomaly_score}'"
 
 
+# This rule detects command evasion via quotes(single/double).
+# For e.g.,
+#
+# 'i'fconfig
+# "e"cho whatever
+# 'i'p a
+#
+# Regular expression generated from regex-assembly/932290.ra.
+# To update the regular expression run the following shell script
+# (consult https://coreruleset.org/docs/development/regex_assembly/ for details):
+#   crs-toolchain regex update 932290
+#
+SecRule REQUEST_COOKIES|REQUEST_COOKIES_NAMES|ARGS_NAMES|ARGS|XML:/* "@rx (?:'[0-9A-Z_a-z]+'|\"[0-9A-Z_a-z]+\")[0-9A-Z_a-z]+\b|[0-9A-Z_a-z]+(?:'[0-9A-Z_a-z]+'|\"[0-9A-Z_a-z]+\")" \
+    "id:932290,\
+    phase:2,\
+    block,\
+    capture,\
+    t:none,\
+    msg:'Remote Command Execution: Quote evasion Found',\
+    logdata:'Matched Data: %{TX.0} found within %{MATCHED_VAR_NAME}: %{MATCHED_VAR}',\
+    tag:'application-multi',\
+    tag:'language-shell',\
+    tag:'platform-unix',\
+    tag:'attack-rce',\
+    tag:'paranoia-level/1',\
+    tag:'OWASP_CRS',\
+    tag:'OWASP_CRS/ATTACK-RCE'\
```

Nope, there needs to be a comma :)
```suggestion
    tag:'OWASP_CRS/ATTACK-RCE,'\
```